dali/test/python/test_dali_cpu_only.py

```diff
@@ -457,6 +498,69 @@ def test_caffe2_reader_cpu():
     for _ in range(3):
         pipe.run()
 
+def test_nemo_asr_reader_cpu():
+    with tempfile.TemporaryDirectory() as tmp_dir:
```

Comment:
note for the future: We should probably add a manifest in DALI_extra.

Reply:
True